canonical-web-and-design/dotrun
# v1.1.5: Access to ~/.npmrc

Unfortunately this will require approval, which I requested here: https://forum.snapcraft.io/t/personal-files-request-access-to-npmrc-to-run-yarn-in-snap/15937

Also added a `clean-cache` command.

Fixes #26
Fixes #29 

QA
--

``` bash
touch ~/.npmrc
sudo snap install --candidate dotrun `# OR` sudo snap refresh --candidate dotrun
sudo snap connect dotrun:dot-npmrc
dotrun exec sh -c 'yarn install && yarn help'
```

^ Check that last command completes without an error.
